### PR TITLE
fix: Add blank check for file param in Import API

### LINKED ISSUE #3031: Import API should throw an error if there is no valid file
Contact Imports API now returns 200 success even if there is no file attached in the request. It should ideally throw a 422. 

To reproduce:

1. POST to `/api/v1/accounts/${accountId}/contacts/import`.
2. API will return a 200 Success. 
3. Look at the logs, it will throw an error in the worker processing.
Screenshot 2021-09-17 at 3 24 17 PM

### PR DESCRIPTION
**Pull request by murtaza-bagwala**
## Description

Fix for Contact Imports API, it returns 200 success even if there is no file attached in the request

Fixes #3031 

## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue).

## How Has This Been Tested?

Tried passing `import_file` param as `nil` in request payload and verified it.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/controllers/api/v1/accounts/contacts_controller.rb [hunk 1]
```diff
@@ -30,10 +30,13 @@ def search
   end
 
   def import
+    render json: { error: 'File is blank' }, status: :unprocessable_entity and return if params[:import_file].blank?
```
Comment (pranavrajs) [inline app/controllers/api/v1/accounts/contacts_controller.rb:33]: @murtaza-bagwala Would you be able to move this to the i18n file? `config/locales/en.yml`?

**Review by pranavrajs — APPROVED** [review]
Nice work! Thanks for the PR. I have a small suggestion. Let us get this merged once that is done.

File: app/controllers/api/v1/accounts/contacts_controller.rb (hunk 1 above)
Comment (pranavrajs) [reply app/controllers/api/v1/accounts/contacts_controller.rb:33]: Also the rubocop is failing, you can see the issues with `rubocop -a` command.

Comment (murtaza-bagwala) [reply app/controllers/api/v1/accounts/contacts_controller.rb:33]: sure @pranavrajs

**Review by pranavrajs — APPROVED** [review]
Looks good to me. @sojan-official Can you please take a look and get this merged?

**Pull request merged**